CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -359,6 +359,15 @@ protected function getTemplateUpdates() {
           ['name' => 'petition_sign', 'type' => 'subject'],
         ],
       ],
+      [
+        'version' => '5.67.alpha1',
```

Comment:
5.68.alpha1, 'cuz global inflation.

Reply:
Lol - we have 12% inflation per year on an ongoing basis!